src/liberror/specification

```diff
@@ -660,3 +660,58 @@ description:python warning
 severity:warning
 ingroup:plugin
 module:python
+
+number:113
+description:couldnt open file.
+severity:error
+ingroup:plugin
+module:lineendings
+
+number:114
+description:invalid line ending.
+severity:error
+ingroup:plugin
+module:lineendings
+
+number:115
+description:inconsistent line endings
+severity:error
+ingroup:plugin
+module:lineendings 
+
+number:116
+description: couldn't open file.
+severity:error
+ingroup:plugin
+module:csvstorage
+
+number:117
+description:invalid number of columns
+severity:error
+ingroup:plugin
+module:csvstorage
+
+number:118
+description:see Reason.
```

Comment:
Thats a bit too generic. Maybe its not necessary. "empty file" could also be reported with 117.
and 117 is a quite good fit for "illegal number of columns" (the other error reported by 118).